apps/condo/domains/banking/hooks/useBankSyncTaskUIInterface.tsx

```diff
@@ -19,7 +21,16 @@ export const useBankSyncTaskUIInterface = () => {
     const BankSyncTaskProgressDescriptionCompleted = intl.formatMessage({ id: 'tasks.BankSyncTask.progress.description.completed' })
     const UpdateTitle = intl.formatMessage({ id: 'Update' })
 
-    const { reload } = useRouter()
+    const { reload, push } = useRouter()
+
+    const onCompleteButtonClick = useCallback((taskRecord) => () => {
```

Comment:
this callback returns handler, so there is naming issue, it should be named something like `getCompleteButtonClickHandler`